### ISSUE TITLE
Sway complains about "popup_during_fullscreen" being an unknown/invalid command

### ISSUE THREAD

**Issue by johnae** [swaywm/sway#3557]
Sway version: `sway version 0c975af1fb701e9a9f0018da0c2f9fdf32234029`

I will get back on this and post a debug log and a minimal config, don't have time now. However I can also just mention that the config I'm using is one I've been using for a long time and just a few days ago when I updated sway, it started complaining that "popup_during_fullscreen" is an unknown/invalid command.

Ofc. I was just setting it to "smart" which is the default according to man pages so perhaps I could just not set it. Still, seems to be something wrong here though.

**Comment by RedSoxFan** [comment]
I think this was unknowingly fixed by #3550. The handlers struct in `sway/commands.c` did not have the commands in alphabetical order in 0c975af, which would have caused issues for bsearch.

Snippet from 0c975af:
```c
{ "popup_during_fullscreen", cmd_popup_during_fullscreen },
{ "pointer_constraint", cmd_pointer_constraint },
```

**Comment by johnae** [comment]
@RedSoxFan Yeah it may have been since I can't reproduce this anymore and I'm on latest sway/wlroots now. Closing this.
